### PR TITLE
suites/rados: add test for 11429

### PR DESCRIPTION
**Pull request by athanatos**
This patch also adds some convenience facilities for making
some of the ceph_manager methods into tasks usable from a
yaml file.

Signed-off-by: Samuel Just <sjust@redhat.com>

### CODE REVIEW

File: tasks/ceph_manager.py [hunk 1]
```diff
@@ -1902,3 +1902,20 @@ def get_filepath(self):
         Return path to osd data with {id} needing to be replaced
         """
         return "/var/lib/ceph/osd/ceph-{id}"
+
+def utility_task(name):
```
Comment (andrewschoen) [inline tasks/ceph_manager.py]: This needs some docs, please.

File: tasks/radosbench.py [hunk 2]
```diff
@@ -59,12 +61,13 @@ def task(ctx, config):
...
-        if config.get('pool'):
-            pool = config.get('pool')
-            if pool is not 'data':
-                ctx.manager.create_pool(pool, erasure_code_profile_name=profile_name)
-        else:
-            pool = ctx.manager.create_pool_with_unique_name(erasure_code_profile_name=profile_name)
+        if config.get('create_pool', True):
+            if config.get('pool'):
+                pool = config.get('pool')
+                if pool is not 'data':
```
Comment (andrewschoen) [inline tasks/radosbench.py]: nitpick: I would prefer != here as we're comparing values not identities.  This should work fine though.

File: tasks/utility.py [hunk 3]
```diff
@@ -0,0 +1,9 @@
+import logging
+import time
+
+log = logging.getLogger(__name__)
+
+def sleep(ctx, config):
```
Comment (jdurgin) [inline tasks/utility.py]: this kind of thing fits well in teuthology.git/teuthology/misc.py

Comment (jdurgin) [inline tasks/utility.py]: I see it's a task. Makes sense as a generic teuthology one imho

**Comment by athanatos** [comment]
I want to retest with the changes, don't merge just yet

**Comment by zmc** [comment]
This looks good to me. You mentioned you wanted to test it one more time, so feel free to do so if it works!

**Pull request merged**